examples/src/main/scala/examples/scala/io/Json.scala

```diff
@@ -0,0 +1,47 @@
+package examples.scala.io
```

Comment:
a separate example might not be needed for every format, but it will be great if this can be converted to a bunch of unit tests instead. current coverage is 0 and is the next thing on my agenda :)